docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Messaging_Connect_a_pooled-connection-factory_to_a_Remote_Artemis_Server.adoc

```diff
@@ -32,6 +32,13 @@ created at step (2).
 /subsystem=messaging-activemq/pooled-connection-factory=remote-artemis:add(connectors=[remote-artemis], entries=[java:/jms/remoteCF])
 ----
 
+Please note that if you are connecting to an Artemis 2.x the old prefixes may not be used anymore for queues and topics.
```

Comment:
1) This should explain what's meant by these prefixes, an example or something. It's not covered earlier in the document.
2) This should state that by default WildFy will prepend the prefix.